# ES6 Import Vertical Alignment (es6-import-vertical-alignment)

## Rule Details

This rule aims to vertically align groups of ES6 import statements. The
alignment anchor is the 'from' in the statement.

Examples of **incorrect** code for this rule:

```js

import module from 'module1'
import anotherOneHere from 'module2'
import yetAnother from 'module3'

```

Examples of **correct** code for this rule:

```js

import module         from 'module1'
import anotherOneHere from 'module2'
import yetAnother     from 'module3'

```
